frontend/src/common/localStorage/get.ts

```diff
@@ -1,5 +1,7 @@
+import { isSSR } from "../utils/isSSR";
```

Comment:
Since Gatsby runs Server Side Rendering (SSR) to generate static files, our code needs to be aware of SSR and acts accordingly